addons/mail/wizard/mail_compose_message.py

```diff
@@ -395,18 +395,18 @@ def get_mail_values(self, res_ids):
         for res_id in res_ids:
             # static wizard (mail.message) values
             mail_values = {
-                'subject': self.subject,
-                'body': self.body or '',
-                'parent_id': self.parent_id and self.parent_id.id,
-                'partner_ids': [partner.id for partner in self.partner_ids],
                 'attachment_ids': [attach.id for attach in self.attachment_ids],
                 'author_id': self.author_id.id,
+                'body': self.body or '',
                 'email_from': self.email_from,
-                'record_name': self.record_name,
-                'reply_to_force_new': self.reply_to_force_new,
                 'mail_server_id': self.mail_server_id.id,
                 'mail_activity_type_id': self.mail_activity_type_id.id,
                 'message_type': 'email' if mass_mail_mode else self.message_type,
+                'parent_id': self.parent_id and self.parent_id.id,
+                'partner_ids': [partner.id for partner in self.partner_ids],
```

Comment:
same here (`self.partner_ids.ids` or `list(self.partner_ids.ids)` if it don't work)